tests/installation/accept_license.pm

```diff
@@ -20,9 +20,15 @@
 use strict;
 use base "y2logsstep";
 use testapi;
+use utils "sle_version_at_least";
 
 sub run {
     my ($self) = @_;
+    # workaround for bsc#1069124: Skip the license check in upgrade mode
+    if (sle_version_at_least('15') && check_var('DISTRI', 'sle') && get_var('UPGRADE')) {
+        record_soft_failure('bsc#1069124');
+        return;
+    }
```

Comment:
How about just

```perl
return record_soft_failure('bsc#1069124 Skip the license check in upgrade mode') if sle_version_at_least('15') && check_var('DISTRI', 'sle') && get_var('UPGRADE');
```

IMHO just as readable but only one line instead of five and giving more information in the soft failure info box

Reply:
Perl style? Sure

Reply:
one-line style :P